app/src/main/java/ar/com/wolox/android/training/ui/training/details/DetailsFragment.kt

```diff
@@ -23,7 +23,7 @@ class DetailsFragment @Inject constructor() : WolmoFragment<DetailsPresenter>(),
 
     override fun layout(): Int = R.layout.fragment_details
 
-    override fun handleArguments(arguments: Bundle?) = arguments != null
+    override fun handleArguments(arguments: Bundle?) = arguments != null && arguments.containsKey(KEY_NEWS_ITEM)
 
     override fun init() {
         val itemExtra = arguments?.getSerializable(KEY_NEWS_ITEM)
```

Comment:
Since you're using the `handleArguments` you can assert the arguments are not null. So use `arguments!!`.